lightningd/bitcoind.c

```diff
@@ -784,89 +784,88 @@ void bitcoind_getchaininfo_(struct bitcoind *bitcoind,
 			    req);
 }
 
-struct get_output {
+/* `gettxout`
```

Comment:
Can we call this `getutxout` to emphasize that it's only UTXOs?

Reply:
Done.